tests/test_Collection.py

```diff
@@ -499,4 +499,14 @@ def test_DisplaySystem():
                [.1,.1,.1], # float
                b.position] # float64
 
-    a.displaySystem(markers,suppress=False, direc=True, subplotAx=None)
\ No newline at end of file
+    a.displaySystem(markers,suppress=False, direc=True, subplotAx=None)
```

Comment:
Like we discussed in the office, we shouldn't test `suppress=False` because this results in `pyplot.show()` being called and opening windows while tests occur (which makes it a pain to test the library if we run the test suite while working on a feature).